public/server/db/lmao.json

```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "Laughing My Ass Out"
```

Comment:
Is `Laughing my ass off` not more common?
```suggestion
  "definition": "Laughing My Ass Off"
```

Reply:
You're right. "off" is more commonly used. I should add "out" as an alternative.